src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -36,16 +36,16 @@
     }
 
     // Identifies the consumer
-    private String uid = UUID.randomUUID().toString();
+    private final String uid = UUID.randomUUID().toString();
 
     private MessageConsumer<String> uidMessageConsumer;
 
     // The queues this verticle is listening to
-    private Map<String, QueueState> myQueues = new HashMap<>();
+    private final Map<String, QueueState> myQueues = new HashMap<>();
     
-    private Map<String, AtomicInteger> myQueueFailureCounts = new HashMap<>();
+    private final Map<String, Integer> myQueueFailureCounts = new HashMap<>();
```

Comment:
`AtomicInteger` is not used because we're single threaded anyways. So there's no situation where multiple threads could access the failure count?

Reply:
Not even the underlying HashMap is thread-safe...
And yes: we are a Verticle - so we are single threaded.
Note also that we typically only have small values in this Map (1, 2, 3, ....). `java.lang.Integer` can handle values -128 ... +127 quite efficient by _not_ creating new objects (except when you use `new Integer(...)` - which you shouldn't